internal/cmd/scan/scan.go

```diff
@@ -150,6 +154,8 @@ For example, if there is a "go.mod" in the target path, its dependencies are goi
 			"Example: debricked resolve --prefer-npm",
 		}, "\n")
 	cmd.Flags().BoolP(NpmPreferredFlag, "", npmPreferred, npmPreferredDoc)
+	cmd.Flags().BoolVar(&sbom, SBOMFlag, false, `Toggles wether to generate and download an SBOM report after scan completion`)
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&sbom, SBOMFlag, false, `Toggle whether to generate and download an SBOM report after scan completion`)
```

or "Toggle generation and download of SBOM report after scan completion"